index.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
```

Comment:
lets move it out of index.js and into another file so we have one file per data structure